IFC4x3/PropertyConstants/o/OCL_WITH_CATENARY_SUSPENSION_1lidkbH2b6qfwnFrj5vFjT.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="OCL_WITH_CATENARY_SUSPENSION_1lidkbH2b6qfwnFrj5vFjT" Name="OCL_WITH_CATENARY_SUSPENSION" UniqueId="6fb27ba5-4429-46d2-9eb1-3f5b45e4fb5d">
```

Comment:
The ones without documentation could be deleted, right? Same for below.

Reply:
This one too, and others below: https://github.com/bSI-InfraRoom/IFC-Specification/pull/190/commits/be731fb1169a876200741df2c81db4c91e730337